Nexpo.Tests/Controllers/EventsControllerTest.cs

```diff
@@ -247,5 +247,23 @@ public async Task PutEmpty()
             Assert.True(responseObject.Start == "08:15", $"Description was actually ({responseObject.Start})");
             Assert.True(responseObject.Location == "Cornelis", $"Description was actually ({responseObject.Location})");
         }
+
+        [Fact]
+        public async Task TestFoodPref()
+        {
+            var application = new WebApplicationFactory<Nexpo.Program>();
+            var client = application.CreateClient();
+            var token = await Login("company", client);
+
+            var response = await client.GetAsync("/api/events/2/foodprefs");
+
+            Assert.True(response.StatusCode.Equals(HttpStatusCode.OK), response.StatusCode.ToString());
```

Comment:
This test should be extended by also checking the content of the returned EventFoodDto since the StatusCode may be OK even when the wrong content is returned.